src/css/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$header-font-color: #5C75DA;
+$body-background-color: #EAEBEF;
+$data-card-background-color: #FEFEFE;
+
+$margin-none: 0;
+$margin-5px: 5px;
```

Comment:
This variable name is a little too specific. If you ever decide that instead of wanting 5px margins everywhere, you want 6px margins, now you have to change the variable name AND the value AND all the areas where you're using that variable name. We want these variable names to be just a little more generic so that we don't have to change them as our design changes.